_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,190 @@
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]
+- [G186: Using a control in the Web page that stops moving, blinking, or auto-updating content][g186]
+- [F16: Failure of Success Criterion 2.2.2 due to including scrolling content where movement is not essential to the activity without also including a mechanism to pause and restart the content][f16]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The text content automatically updates after the page completes loading. A button is available to stop the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="stopUpdates()" value="Stop updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 2
+
+The text content automatically updates after the page completes loading. A button is available to pause and resume the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="toggleUpdates()" value="Pause updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 3
+
+The text content automatically updates after the page completes loading. A button is available to hide the automatically updating content.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="hide()" value="Hide updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The text content automatically updates after the page completes loading. There is no component to stop or pause the automatic updates or to hide the updating content.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Inapplicable
```

Comment:
For coverage of the  condition, we need inapplicable examples with:
- no auto-updating content;
- children whose content also changes;
- changes happening before the readiness is "complete".

For the children case, that might maybe not be inapplicable because then one of the children whose content changes is going to be applicable. So likely a passed example where the description says that the top-level element is not applicable because of its child changing, but the child is applicable and passing.